system/security/LdapSecurity/ldapconnection.cpp

```diff
@@ -1431,7 +1431,7 @@ class CLdapClient : implements ILdapClient, public CInterface
             createLdapBasedn(NULL, m_ldapconfig->getResourceBasedn(RT_DEFAULT), PT_ADMINISTRATORS_ONLY);
             createLdapBasedn(NULL, m_ldapconfig->getResourceBasedn(RT_FILE_SCOPE), PT_ADMINISTRATORS_ONLY);
             createLdapBasedn(NULL, m_ldapconfig->getResourceBasedn(RT_VIEW_SCOPE), PT_ADMINISTRATORS_ONLY);
-            createLdapBasedn(NULL, m_ldapconfig->getResourceBasedn(RT_WORKUNIT_SCOPE), PT_ADMINISTRATORS_ONLY);
+            createLdapBasedn(NULL, m_ldapconfig->getResourceBasedn(RT_WORKUNIT_SCOPE), PT_DEFAULT);//Include "Authenticated Users" to enable default behavior
```

Comment:
This scope is different from the others in that it is the only one with a "default" setting to be applied when there are no scopes defined for a given user. By granting access to authenticated users (the default behavior), users will be able to run workunits out of the box.  The alternative is that an admin would have to create a scope for each and every user.
That being said, our LDAP Admins have stated that we cannot grant Authenticated Users access to any OU, so I am cancelling this PR and will investigate an alternative approach